test-case/check-pause-resume.sh

```diff
@@ -116,13 +118,30 @@ expect {
         if { \$i > $repeat_count } { exit 0 }
         exp_continue
     }
+    -re ".*\:.*\n"|"Hardware PCM card.*\n" {
+        puts "\rexpect ignore dump-hw-params"
+        after 10
+        exp_continue
+    }
+# NOTE: we could use 'defatult' = timeoue + eof but eof is not the obvious problem.
+#       So handle timeout specifically
```

Comment:
I don't understand. EOF means arecord died, how is that not a problem? Test with a wrong PCM and you should see.

Reply:
Glad I put my comments there. I thought eof didn't make sense for this test, but I think good point. Let me do some test for the case.
I have faced all the comments I could expect. haha :)

Reply:
I see typo, defatult -> default. WIll handle in next patch